tests/integration_test.rs

```diff
@@ -121,3 +122,73 @@ fn test_show_source() -> Result<()> {
     assert!(str::from_utf8(&output.get_output().stdout)?.contains("l = 1"));
     Ok(())
 }
+
+#[test]
+fn test_ruff_black_compatibility() -> Result<()> {
+    let fixtures_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+        .join("resources")
+        .join("test")
+        .join("fixtures");
+
+    let paths: Vec<walkdir::DirEntry> = WalkDir::new(fixtures_dir)
+        .into_iter()
+        .filter(|entry| {
+            entry.as_ref().map_or(true, |entry| {
+                let file_name = entry.path().to_string_lossy();
+                file_name.ends_with(".py")
+            })
+        })
+        .filter_map(|entry| entry.ok())
+        .collect();
+
+    let ruff_args = [
+        "-",
+        "--silent",
+        "--exit-zero",
+        "--fix",
+        "--line-length",
+        "100",
+        "--select",
+        "A,B,C,D,E,F,I,N,PGH,PLC,PLE,PLR,PLW,Q,RET,S,SIM,T,UP,W,YTT",
```

Comment:
Maybe use this instead.
```suggestion
        CheckCategory::iter()
            .map(|category| category.codes().iter().map(AsRef::as_ref).join(", "))
            .join(", ")
```

Reply:
This is fantastic, thank you. :)